trigger/patch.go

```diff
@@ -152,6 +154,10 @@ func (t *patchTriggers) makeData(sub *event.Subscription) (*commonTemplateData,
 			Color:     slackColor,
 		})
 	}
+	finishTime := t.patch.FinishTime
+	if utility.IsZeroTime(finishTime) {
+		finishTime = time.Now()
```

Comment:
is this really the finish time, though? I would have thought the finish time is the time the last task in the patch that actually ran finished running.

Reply:
my intent is that this would approximate the finish time, assuming the trigger is processed no more than a minute after the patch ends. I've changed it to be something more accurate though